src/VRCFaceTracking.Avalonia/Views/MainWindow.axaml.cs

```diff
@@ -12,6 +16,36 @@ public MainWindow(MainViewModel vm)
     {
         DataContext = vm;
         InitializeComponent();
+        AdjustTitleBarForPlatform();
+    }
+
+    private void AdjustTitleBarForPlatform()
```

Comment:
Whenever we have platform specific code, it's preferable to use the OS preprocessors we have defined for WINDOWS_DEBUG/RELEASE, MACOS_DEBUG/RELEASE, LINUX_DEDUG/RELEASE.

As for iOS and Android, this can be left as it is. I imagine we'll need to add a preprocessor block for them at some point

Reply:
Can definitely work with preprocessors.